prep-javascript-dom-events/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Events</title>
+</head>
+
+<body>
+  <button class="hello-world-btn">Hello World!</button>
+  <script>
+    document.addEventListener('DOMContentLoaded', function () {
+  console.log('content loaded!');
+});
+    window.addEventListener('mousemove', function () {
+  console.log('mouse is moving!');
+});
+    var $helloWorld = document.querySelector('.hello - world - btn');
```

Comment:
If you remove the spaces in your selector, your click listener should start working.

```js
var $helloWorld = document.querySelector('.hello-world-btn');
```